arwen/host/execution_test.go

```diff
@@ -346,10 +346,10 @@ func TestExecution_ExecuteOnSameContext_Simple(t *testing.T) {
 	input.GasProvided = gasProvided
 
 	vmOutput, err := host.RunSmartContractCall(input)
-	fmt.Println(vmOutput.ReturnMessage)
 	require.Nil(t, err)
 	require.Equal(t, vmcommon.Ok, vmOutput.ReturnCode)
 	require.Equal(t, "", vmOutput.ReturnMessage)
+	fmt.Println(vmOutput.ReturnMessage)
```

Comment:
maybe delete this altogether?

Reply:
left it as it is, maybe needed for debug

Reply:
use log then